Utilities.Rest/Packages/com.utilities.rest/Runtime/Rest.cs

```diff
@@ -983,6 +983,67 @@ public static bool TryGetFileNameFromUrl(string url, out string fileName)
             return filePath;
         }
 
+        /// <summary>
+        /// Download a file from the provided <see cref="url"/> and return the <see cref="byte[]"/> array of its contents.
+        /// </summary>
+        /// <param name="url">The url to download the file from.</param>
+        /// <param name="fileName">Optional, file name to download (including extension).</param>
+        /// <param name="parameters">Optional, <see cref="RestParameters"/>.</param>
+        /// <param name="debug">Optional, debug http request.</param>
+        /// <param name="cancellationToken">Optional, <see cref="CancellationToken"/>.</param>
+        /// <returns>The <see cref="byte[]"/> of the downloaded file.</returns>
+        public static async Task<byte[]> DownloadFileBytesAsync(
+        string url,
+        string fileName = null,
+        RestParameters parameters = null,
+        bool debug = false,
+        CancellationToken cancellationToken = default)
+        {
+            await Awaiters.UnityMainThread;
+            byte[] bytes = null;
+
+            var filePath = await DownloadFileAsync(url, fileName, parameters, false, cancellationToken);
```

Comment:
This prob won't work on WebGL, and we should fall back to `DownloadBytesAsync`

Reply:
If you are on WebGL, then you would not use it, so why have a fallback?
WebGL would only use DownloadBytes